docker-compose.yml

```
@@ -2,15 +2,13 @@ version: '3'
 
 services:
   nest-desktop:
-    image: docker-registry.ebrains.eu/nest/nest-desktop:3.1
-    environment:
-      NEST_SIMULATOR_PORT: 52425
+    image: docker-registry.ebrains.eu/nest/nest-desktop:3.2
```

I'd like to suggest to keep them, so that it is more obvious to the user that this can be adjusted:
```suggestion
    image: docker-registry.ebrains.eu/nest/nest-desktop:3.2
    environment:
      NEST_SIMULATOR_PORT: 52425
```